### ISSUE TITLE
Getting `Syntax error or access violation: 1067 Invalid default value for 'ordered_at'` column

### ISSUE THREAD

**Issue by Rijoanul-Shanto** [lmsqueezy/laravel#84]
### Lemon Squeezy for Laravel Version

1.5.2

### Laravel Version

11

### PHP Version

8.2.17

### Description

getting `SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'ordered_at' (Connection: mysql, SQL: create table 'lemon_squeezy_orders'...)` while running the migration.

Database: `MariaDB 10.6.16`

### Steps To Reproduce

1. Add the package
2. Run migration

**Comment by heyjorgedev** [comment]
Did you get this error while running the migrations? If there is no data on the table how is it complaining about not being nullable?

**Comment by driesvints** [comment]
Yeah bit weird this one.. this should definitely come from data being inserted into the table. I see the it's with a create table statement but just super odd...

**Comment by Rijoanul-Shanto** [comment]
@heyjorgedev @driesvints Please refer to the reference below. I have a completely new database with zero tables in it.

![image](https://github.com/lmsqueezy/laravel/assets/17069712/80c99b8c-ba87-45fb-8ddc-289fa4872515)

And after running the migration, here's what it looks like:
![image](https://github.com/lmsqueezy/laravel/assets/17069712/e5b90792-8101-436c-b665-2d1c24920d28)

**Comment by Rijoanul-Shanto** [comment]
> @heyjorgedev @driesvints Please refer to the reference below. I have a completely new database with zero tables in it.
> 
> ![image](https://private-user-images.githubusercontent.com/17069712/322799216-80c99b8c-ba87-45fb-8ddc-289fa4872515.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyNjg1OTIsIm5iZiI6MTcxMzI2ODI5MiwicGF0aCI6Ii8xNzA2OTcxMi8zMjI3OTkyMTYtODBjOTliOGMtYmE4Ny00NWZiLThkZGMtMjg5ZmE0ODcyNTE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDExNTEzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3YjAyMDE1NjljOTgzODczNjAzYWQyMThkNzEwMGYwYzVhOTcyYzBhMjNhMzliMGQwMTQ5MGM2OTI0NmY1N2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.T7_B6THlWeb26IKc7Vp03g-1URXh0CdQ1tRWulxUxtQ)
> 
> And after running the migration, here's what it looks like: ![image](https://private-user-images.githubusercontent.com/17069712/322800132-e5b90792-8101-436c-b665-2d1c24920d28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyNjg1OTIsIm5iZiI6MTcxMzI2ODI5MiwicGF0aCI6Ii8xNzA2OTcxMi8zMjI4MDAxMzItZTViOTA3OTItODEwMS00MzZjLWI2NjUtMmQxYzI0OTIwZDI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDExNTEzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjYTI2Y2M3MzIxMDE0MWM1NDgxNmNjYjk2MGU0MmYyZjBmNmYxZDg4Y2M5MjA3NDIyZjFkMDg2NjM0M2Y2MGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2rEKre8Fhd-7AjzdURyyYY48xgxJvDBuQ88Ev2mPN58)

It seems that the issue isn't occurring with a freshly installed Laravel app. Let me delve deeper into what's causing the actual problem.

**Comment by Rijoanul-Shanto** [comment]
@heyjorgedev @driesvints 

I've tried migrating in both a fresh Laravel app and a fresh MySQL and MariaDB database, and it seems to be related to my previous MariaDB configuration rather than the package. So, I'm closing this issue for now. Thanks for taking the time to reach out. I'll provide an update once I've pinpointed the actual issue. Initially, I thought the DB strict mode might have caused the issue, but I'm not certain yet.
